app/constants/constants.js

```diff
@@ -194,6 +197,28 @@ export const CONNECTION_CONFIG = {
             'type': 'password',
             'description': 'Your data.world read/write token. It can be obtained from https://data.world/settings/advanced'
         }
+    ],
+    [DIALECTS.ATHENA]: [
+        {
+            'label': 'S3 Access Key', 'value': 'accessKey', 'type': 'password'
+        },
+        {
+            'label': 'S3 Secret Access Key', 'value': 'secretKey', 'type': 'password'
+        },
+        {
+            'label': 'AWS Region', 'value': 'region', 'type': 'text'
+        },
+        {
+            'label': 'S3 Bucket', 'value': 's3Outputlocation', 'type': 'text',
+            'description': 'The Athena will put the output results of the query in this location.'
```

Comment:
How about:
```
'description': 'The Athena connector will store query results in this location.'
```
?

Is this something optional? I mean, does Athena always store query results in a bucket?
If optional, please, add a description.

Reply:
It is required.  It is where Athena uses to the results of the output from the Query.  Here is a link to the docs:

https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Athena.html#startQueryExecution-property

Essentially, Athena has a process to start the query and it returns an query id.  Then you  have to periodically check the status of that query id to see if it is completed.  Once it is completed you can call getQueryResults.

Let me know if you have other questions

Reply:
OK, thank you for the explanation. See also the suggestion below about the naming of this property.